processor/tailsamplingprocessor/README.md

```diff
@@ -22,7 +22,7 @@ Multiple policies exist today and it is straight forward to add more. These incl
 - `string_attribute`: Sample based on string attributes (resource and record) value matches, both exact and regex value matches are supported
 - `trace_state`: Sample based on [TraceState](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracestate) value matches
 - `rate_limiting`: Sample based on rate
-- `span_count`: Sample based on the minimum number of spans within a batch. If all traces within the batch have less number of spans than the threshold, the batch will not be sampled.
+- `span_count`: Sample based on the minimum and maximum number of spans. If the sum of all spans in the trace is outside the range threshold, the trace will not be sampled.
```

Comment:
I think it should mention that the boundaries are inclusive. Like, max_spans=20 will return spans with 20, but not 21.

Reply:
UPDATE mentions inclusion of minimum and maximum values